Misc/NEWS.d/next/Documentation/2020-04-28-15-35-12.bpo-40405.ZU8r_9.rst

```diff
@@ -0,0 +1 @@
+Documentation fixes for :func:`asyncio.as_completed`.
```

Comment:
Changes to the documentation do not usually require a NEWS entry.

Reply:
Yeah, I know, but I can't set the 'NO-NEWS' tag.

Reply:
Don't worry about it, the bot will complain but a core developer should do it during the review.

Reply:
@bharel The `skip news` label has been applied, so this can be removed. In the future for any small documentation changes (or other minor changes that don't require a news entry) you can simply omit it, and a core dev or triager will apply the label, if appropriate. It can sometimes help if you explicitly mention that you think the news entry should be skipped.